alacritty.yml

```diff
@@ -145,6 +145,17 @@ font:
     # The `style` can be specified to pick a specific face.
     #style: Italic
 
+  # Bold italic font face
+  #bold_italic:
+    # Font family
+    #
+    # If the bold italic family is not specified, it will fall back to the
+    # value specified for the normal font.
+    #family: monospace
+
+    # The `style` can be specified to pick a specific face.
+    #style: Bold Italic
```

Comment:
Just as a note, I thought that `Bold Italic` might not be the correct thing here and instead maybe `BoldItalic` or something entirely different would be required to make things work. However, after testing it with `Fira Sans` (since `Fira Mono` does not have this variant), this does work properly and `BoldItalic` does not.

So looking good like this.

Reply:
Yeah, just as the normal font face is conventionally “Regular”, this one is conventionally “Bold Italic”.